.github/actions/transfer/action.yml

```diff
@@ -52,7 +52,7 @@ runs:
         commands: ${{ env.COMMANDS }}
     - name: "Compose rsync exclude flags"
       shell: bash
-      run: |
+      run: >
```

Comment:
als ik de excluding paths in een variable steek dan execute hij deze lijn per lijn
hier werkt het niet: https://github.com/esign/uptime-robot/actions/runs/4253009283/jobs/7397303451
hier wel: https://github.com/esign/uptime-robot/actions/runs/4232702946/jobs/7352756669